test/integration/test_integration_basics.py

```diff
@@ -109,6 +114,30 @@ def test_http_h2_mini_stress_test_without_client_side_queueing(http_test_server_
   assertNotIn("upstream_rq_pending_overflow", counters)
 
 
+def test_http_h1_mini_stress_test_open_loop(http_test_server_fixture):
+  """
+  H1 open loop stress test. We expect higher pending and overflow counts 
+  """
+  counters = mini_stress_test(http_test_server_fixture, [
+      http_test_server_fixture.getTestServerRootUri(), "--rps", "2500", "--max-pending-requests",
+      "1", "--duration 10", "--open-loop", "--max-active-requests", "1", "--connections", "1"
```

Comment:
Nit: I think you split all the others yesterday to `"--duration", "10"`

Reply:
Fixed in 0fcba50e4bbd0a645251ec81a9e377562aa433e6